lib/form1010_ezr/service.rb

```diff
@@ -60,24 +64,22 @@ def submit_form(parsed_form)
       log_and_raise_error(e, parsed_form)
     end
 
-    def log_submission_failure(parsed_form)
-      StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.failed_wont_retry")
+    def log_submission_failure(parsed_form, retries_exhausted: false)
```

Comment:
So it looks like this method is being used for both when an ezr submission attempt as failed and after an ezr has submitted 14 times and failed every time, and thus it will not retry any more? I like having different StatsD increments for those two scenarios, as well as the separate logs. I wonder if this logic would be clearer if we just made two distinct methods that are similar but log the different failure paths. So one is `log_submission_failure` that would get logged each time a submission fails, and another is `log_total_submission_failure` (naming is hard so I'm open to better names) or something that shows that the submissions will not be attempted any more. And that is the one that our monitors would probably key off of? That is currently our `failed_wont_retry`.

Reply:
According to Alex, Patrick wants us to keep monitoring in place for retrying failures as well as exhausted ones. I can go ahead and make two separate methods.

Reply:
Sounds good. I agree that I think that's valuable. I just think it will make it clearer which type of error we are logging to have separate methods.

Reply:
Yes, I totally agree! I'll push up the new code in a second 😄